packages/server/modules/emails/services/verification/request.ts

```diff
@@ -184,25 +183,11 @@ export const requestEmailVerificationFactory =
     await sendVerificationEmailFactory(deps)(newVerificationState)
   }
 
-/**
- * Listen for user:created events and trigger email verification initialization
- */
-export const initializeVerificationOnRegistrationFactory =
-  (deps: {
-    userEmitterListener: typeof UsersEmitter.listen
-    requestEmailVerification: RequestEmailVerification
-  }) =>
-  () => {
-    return deps.userEmitterListener(UsersEvents.Created, async ({ user }) => {
```

Comment:
This was the redundant one. Upon adding multiple emails we added another way of triggering email verification directly in the user creation service and both of these were firing.